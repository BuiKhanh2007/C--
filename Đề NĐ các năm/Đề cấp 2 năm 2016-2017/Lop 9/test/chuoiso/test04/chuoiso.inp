6362 123
579
814
428
793
840
115
568
871
553
447
928
65
683
389
9
164
513
459
744
72
669
676
211
302
543
730
629
168
478
18
29
376
754
141
501
609
709
540
251
531
439
264
55
816
134
429
463
923
195
569
218
802
599
433
937
448
200
929
290
734
339
328
575
881
971
952
452
708
332
745
956
702
640
510
377
586
920
216
250
37
772
775
798
251
124
407
269
208
595
838
298
649
657
396
563
29
223
435
245
604
235
699
479
746
365
462
972
551
300
511
181
126
17
249
318
998
471
398
937
101
583
622
302
