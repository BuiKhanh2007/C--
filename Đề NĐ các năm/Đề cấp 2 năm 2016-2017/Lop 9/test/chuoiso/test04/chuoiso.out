1
5
1
2
2
1
1
1
3
1
1
1
2
1
1
2
8
1
8
1
9
1
1
4
1
7
1
5
1
5
1
2
2
4
9
1
8
1
0
5
1
7
4
7
1
6
7
1
5
0
3
7
8
1
9
5
7
7
2
3
2
1
1
8
4
7
1
0
1
1
5
0
8
1
7
1
8
4
2
3
1
0
1
0
8
1
1
1
0
5
5
9
4
1
9
1
0
1
8
9
1
1
7
5
8
2
1
3
7
2
1
5
1
9
8
1
1
3
9
5
9
0
4
